docs/MO_DG/prepare_model/convert_model/Convert_Model_From_PyTorch.md

```diff
@@ -1,11 +1,79 @@
 # Converting a PyTorch Model {#openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_PyTorch}
 
-The PyTorch framework is supported through export to the ONNX format. In order to optimize and deploy a model that was trained with it:
+This page provides instructions on how to convert a model from the PyTorch format to the OpenVINO IR format using Model Optimizer.
+Model Optimizer Python API allows the conversion of PyTorch models using the `convert_model()` method.
+
+## Converting a PyTorch model with MO Python API
+
+Example of PyTorch model converting:
+```sh
+import torchvision
+import torch
+from openvino.tools.mo import convert_model
+
+model = torchvision.models.resnet50(pretrained=True)
+ov_model = convert_model(model)
+```
+
+Following PyTorch model formats are supported:
+* `torch.nn.Module`
+* `torch.jit.ScriptModule`
+* `torch.jit.ScriptFunction`
+
+Converting of some PyTorch models may require model tracing which need setting of `input_shape` or `example_input` parameters.
+
+`example_input` is used as example input for model tracing.
+`input_shape` is used for constructing a float zero-filled torch.Tensor for model tracing.
+
+Example:
+```sh
+import torchvision
+import torch
+from openvino.tools.mo import convert_model
+
+model = torchvision.models.resnet50(pretrained=True)
+ov_model = convert_model(model, example_input=torch.zeros(1, 3, 100, 100))
+```
+
+`example_input` accepts the following formats:
+
+* `openvino.runtime.Tensor`
+* `torch.Tensor`
+* `np.ndarray`
+* `list` or `tuple` with tensors (`openvino.runtime.Tensor` / `torch.Tensor` / `np.ndarray`)
+* `dictionary` where key is the input name, value is the tensor (`openvino.runtime.Tensor` / `torch.Tensor` / `np.ndarray`)
+
+If PyTorch model is scripted it can be converted using `convert_model()` with `input_signature` parameter for keeping dynamic dimensions of the original model.
```

Comment:
it is not correct, input_signature has no influence on dynamic signature (it is just additional names for input tensors aligned with argument names in model forward method)

Reply:
Corrected input_signature  description. Please take a look.